app/scripts/models/reliers/oauth.js

```diff
@@ -160,6 +180,8 @@ define([
           // server version always takes precedent over the search parameter
           self.set('redirectUri', serviceInfo.redirect_uri);
           self.set('origin', Url.getOrigin(serviceInfo.redirect_uri));
+          // parse resume token to set additional properties for the relier
+          self._parseResumeToken();
```

Comment:
@vladikoff The resume token fields don't currently overlap with the client info, but perhaps we should move it before this call (somewhere near search param import) to be safe. I don't see any immediate security/phishing threat but it seems like a possible candidate for one if we're not careful.